includes/admin/settings/class-settings-advanced.php

```diff
@@ -153,6 +153,29 @@ public function get_settings() {
 								'disabled' => __( 'Disabled', 'give' ),
 							],
 						],
+						[
+							'name'    => __( 'Setup Page', 'give' ),
+							/* translators: %s: about page URL */
+							'desc'    => sprintf(
+								wp_kses(
+									__( 'Disable this option if you would like to disable the <a href="%s" target="_blank">GiveWP Setup page</a> that displays when GiveWP is first isntalled.', 'give' ),
+									[
+										'a' => [
+											'href'   => [],
+											'target' => [],
+										],
+									]
+								),
+								esc_url( admin_url( 'edit.php?post_type=give_forms&page=give-setup' ) )
+							),
+							'id'      => 'setup_page_enabled',
+							'type'    => 'radio_inline',
+							'default' => \Give\Onboarding\Onboarding::isSetupPageEnabled() ? 1 : 0,
+							'options' => [
+								1 => __( 'Enabled', 'give' ),
```

Comment:
@kjohnson we are following a standard to define checkbox and radio setting: https://github.com/impress-org/givewp/blob/develop/includes/admin/settings/class-settings-gateways.php#L89. Can we use them instead?

Reply:
Ah, I may have been looking at an older/bad example. Thank you for pointing this out!

Reply:
Updated to match the enabled/disabled convention.